maps/ministation/ministation_objects.dm

```diff
@@ -14,6 +19,35 @@
 		if(!.)
 			. = SSmaterials.weighted_minerals_rich
 
+/turf/exterior/wall/random/ministation/Initialize(var/ml, var/materialtype, var/rmaterialtype)
+	..(ml, materialtype, rmaterialtype)
+	if(ml)
+		starting = TRUE
+	. = INITIALIZE_HINT_LATELOAD
+
+/turf/exterior/wall/random/ministation/LateInitialize()
+	if(starting) //only need to call generate_asteroid here if this is a first node, the others are called in the generate_asteroid function
```

Comment:
Why not only lateload on mapload and remove the starting var entirely?

Reply:
/turf/exterior/wall/ has it's own lateload logic that i didn't want to interfere with.

Reply:
For what it's worth, `LateInitialize` is called with the same args as `Initialize`, including `mapload` as the first parameter.